server/src/main/java/org/apache/iotdb/db/engine/storagegroup/DataRegion.java

```diff
@@ -1727,8 +1731,11 @@ public void readUnlock() {
 
   /** lock the write lock of the insert lock */
   public void writeLock(String holder) {
+    final long startTime = System.nanoTime();
```

Comment:
I suspect that the metric added here does not reflect the formal write request locking time, since the method is likely to be called by a lot of code, perhaps we can just add monitoring to a few calls in the red box.

<img width="1093" alt="image" src="https://user-images.githubusercontent.com/32640567/216798640-d4100279-6c29-445c-b503-6f72e2cf3aed.png">